modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -61,7 +61,7 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 +
 [NOTE]
 ====
-If you do not control the DNS server, you can add the record to your `/etc/hosts` file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
+If you do not control the DNS server, you can add the record to your `/etc/hosts` file. The `/etc/hosts` file does not consider wildcard entries, you need a list with the individual URLs for your applications in the `/etc/hosts` file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
```

Comment:
s/add the record/add the records/

You may want to consider adding a couple example records, like console, oauth, prometheus and grafana (see [this list](https://github.com/shiftstack/shiftstack-ci/blob/7427222b43bc5f4bbdaf5515c28dac99d5d2d584/run_ocp.sh#L138) for reference)

Reply:
I find it a bit confusing here:

the individual URLs **for your applications** in the `/etc/hosts` file. This action makes **the API** accessible to only you

Consider making it explicit that the user needs to add records both for accessing the API, and for accessing their apps (and optionally, for accessing the console).

Reply:
Addressed